main/main.cpp

```diff
@@ -4,20 +4,20 @@
 
 #include "sdkconfig.h"
 #include "esp_system.h"
-#include "esp_event_loop.h"
```

Comment:
This include is required for the call  to ```esp_event_loop_init``` in line 33

Reply:
The contents of `esp_event_loop.h` is:
```c++
#pragma once
#warning "esp_event_loop.h is deprecated, please include esp_event.h instead"
#include "esp_event.h"
```
So I thought it was save to alter it 😉

Reply:
Solved, as we move to the release/v4.0 version 😄